crates/gitbutler-stack/src/stack_branch.rs

```diff
@@ -140,12 +140,30 @@ impl StackBranch {
             .map(|ref_name| ref_name.remote().to_owned())
             .unwrap_or(default_target.push_remote_name());
 
+        let predecessor_branch = stack
+            .branch_predacessor(self)
+            .map(|p| p.name.to_owned())
+            .unwrap_or(default_target.branch.branch().to_owned());
+
+        let predecessor_ref = format!("refs/remotes/{}/{}", remote, predecessor_branch);
```

Comment:
Why not use https://github.com/gitbutlerapp/gitbutler/blob/ea61bf06b4636f424aef8efa1af1f539bc6878e4/crates/gitbutler-stack/src/stack_branch.rs#L86 ?

Reply:
Because that takes a remote name and gives back a ref for the current branch. We need the remote ref for the previous branch.